src/windows.rs

```diff
@@ -78,13 +77,7 @@ impl MmapInner {
         }
     }
 
-    /// Check if a file needs to be writable for copy-on-write mode.
-    #[inline]
-    pub fn needs_write_for_copy() -> bool {
```

Comment:
No, actually, this method isn't used anymore, because we require for the file to be opened in advance.